src/collinearity_tool/collinearity_tool.py

```diff
@@ -47,18 +53,19 @@ def corr_heatmap(df):
     
 def vif_bar_plot(x, y, df, thresh):
```

Comment:
```suggestion
def vif_bar_plot(x, y, df, thresh, intercept=False):
```
This is to go with the intercept option